pip/pip-316.md

```diff
@@ -0,0 +1,83 @@
+
+# PIP-316: Create a producerName field for DeadLetterPolicy
+
+
+# Motivation
+
+Add producerName for dead letter producer, easier to locate problems.
```

Comment:
We need to state the motivation for adding the producer name. Is it only for debugging purposes?

Reply:
It's not just for debugging, it would also be more convenient for monitoring if there was a specific name, but right now we have no idea what this producer actually does.